lib/vmdb/loggers.rb

```diff
@@ -29,10 +29,13 @@ def self.apply_config(config)
       Vmdb::Plugins.each { |p| p.try(:apply_logger_config, config) }
     end
 
-    def self.create_logger(log_file_name, logger_class = ManageIQ::Loggers::Base)
-      create_container_logger(log_file_name, logger_class) ||
-        create_journald_logger(log_file_name, logger_class) ||
-        create_file_logger(log_file_name, logger_class)
+    def self.create_logger(log_file, logger_class = ManageIQ::Loggers::Base)
```

Comment:
Unrelated to this PR, but I'm thinking we should have a way to pass a progname specifically rather than have it based on the file name if we expect to have `IO` as an option here.